tests/webp-uploads-test.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Tests for webp-uploads module.
+ *
+ * @package performance-lab
+ */
+
+class WebP_Uploads_Tests extends WP_UnitTestCase {
+
+	/**
+	 * Test if webp-uploads is loaded and filter applied when system supports WebP.
+	 */
+	function test_webp_uploads_is_loaded() {
+		// Test before enabling.
+		$output_format = apply_filters( 'image_editor_output_format', array(), '', 'image/jpeg' );
+		$this->assertEquals( array(), $output_format );
+
+		// Activate the module.
+		$new_value = array(
+			'webp-uploads' => array( 'enabled' => true ),
+		);
+		update_option( PERFLAB_MODULES_SETTING, $new_value );
+		perflab_load_active_modules();
```

Comment:
Because of how the test suite works right now, this makes sense here, but really we shouldn't require every module test to load its module code.

I think it would make more sense for the test suite to ensure that, for tests specifically, all module files are loaded right away. I already thought about that the other day, so I can look into adding that change here. Then the test here only needs to make sure to add the filter with the function.

Reply:
@adamsilverstein I just added this change here https://github.com/WordPress/performance/pull/30/files#diff-96a82592013e5f4b91e682332583b7a4a6ca1ff0431d267c61179b1a21b167daR40-R55.

Can you copy those exact changes into this PR so that it applies here too? (Alternatively wait for the other PR to be merged, but by doing it here, we can decouple the two.) Once you have that code, you should be able to remove the lines here, since the module code will be loaded right away.

Reply:
👍🏼  makes sense